packages/vite/src/node/watch.ts

```diff
@@ -10,6 +13,7 @@ export function resolveChokidarOptions(
       '**/.git/**',
       '**/node_modules/**',
       '**/test-results/**', // Playwright
+      path.relative(config.root, config.cacheDir + '/**'),
```

Comment:
I think we need to escape glob chars in `config.cacheDir` by using [`fg.escapePath`](https://github.com/mrmlnc/fast-glob#escapepathpattern) or something simillar.

Also I think we don't need to use `path.relative` because `ignored` accepts absolute paths.
https://github.com/paulmillr/chokidar#path-filtering